### PR TITLE
WW-4503 Build Struts with JDK 7

### PR DESCRIPTION
**Pull request by swiftelan**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -352,7 +353,7 @@
             <dependency>
                 <groupId>org.apache.struts</groupId>
                 <artifactId>struts-annotations</artifactId>
-                <version>1.0.5</version>
+                <version>1.0.6-SNAPSHOT</version>
```
Comment (lukaszlenart) [inline pom.xml]: Do we need this? I thought after dropping APT we can drop this as well

**Comment by lukaszlenart** [comment]
It's better to use two dedicated PRs where each addresses just one issue.

File: pom.xml (hunk 1 above)
Comment (lukaszlenart) [inline pom.xml]: Now I get it, we must first update struts-annotations project, release it and then we can use it here

**Comment by lukaszlenart** [comment]
This PRs mixed fixes for two different things :(

**Comment by swiftelan** [comment]
I understand. I merged changes from the GitHub mirror to my fork. I didn't know that it would update the existing pull request, that was not my intention.

**Comment by lukaszlenart** [comment]
Yeah... each change must be in separated branch and every commit to that branch will be automatically exposed as a part of PR. That's how it works :)

**Comment by lukaszlenart** [comment]
Are you going to open a new PR to introduce JDK7? If not I can do it directly based on your changes - it's an easy fix :)

**Comment by swiftelan** [comment]
Moving to Java 7 requires the new annotation parser or the removal of the maven-apt-plugin. The annotation parser [job](https://builds.apache.org/view/S-Z/view/Struts/job/struts-annotations/) is disabled. Using a snapshot version of the annotation processor will fail the build. It might be easier for you to coordinate changes to both repositories.

**Comment by lukaszlenart** [comment]
Ok, I will handle this based on your work, thanks! It will take some time as releasing new version of struts-annotations requires a vote

**Pull request closed without merging**